waiter/src/waiter/settings.clj

```diff
@@ -288,11 +289,13 @@
                                   "scale-down-factor" 0.001
                                   "scale-factor" 1
                                   "scale-up-factor" 0.1}
+   :service-description-resource-limits {"cpus" 100
+                                         "mem" (* 1024 1024)}
```

Comment:
These seem too high for the default maximums

Reply:
Changed to `32` and `128` Gb